src/gui/DatabaseWidget.cpp

```diff
@@ -1750,14 +1750,14 @@ void DatabaseWidget::processAutoOpen()
         QFileInfo filepath;
         QFileInfo keyfile;
 
-        if (entry->url().startsWith("file://")) {
-            QUrl url(entry->url());
+        if (entry->displayUrl().startsWith("file://")) {
```

Comment:
You don't want to use the displayUrl(). You want to pass url() to Entry::resolveMultiplePlaceholders(...)

Reply:
I made changes to use `resolveMultiplePlaceholders`